src/Components/test/E2ETest/ServerExecutionTests/InteropReliabilityTests.cs

```diff
@@ -492,7 +492,7 @@ public async Task DispatchingEventsWithInvalidEventHandlerId()
             Assert.Contains(
                 Logs,
                 e => e.EventId.Name == "DispatchEventFailedToDispatchEvent" && e.LogLevel == LogLevel.Debug &&
-                     e.Exception is ArgumentException ae && ae.Message.Contains("There is no event handler associated with this event. EventId: '1'."));
+                     e.Exception is ArgumentException ae && ae.Message.Contains("There is no event handler associated with this event. EventId: '99999'."));
```

Comment:
Out of interest, could you clarify why you're changing this?

Reply:
Event handler 1 was updated in the old situation and disposed, which is why in this case there was no event handler associated with event handler 1. In the new code, event handler 1 was just updated, and 1 is still in use. Makes no sense to test with id 1, if it is in use, which is why I changed it to 99999 to make sure that it is an invalid eventhandler Id. It took me quite some debugging to convince myself that this was actually the case, but feel free to redo my work if you want to ;-)

Reply:
(The test failed while trying to cast the MouseEventArgs instance onto an eventhandler that clearly was not expecting a mouseeventargs).